src/main/java/uk/gov/hmcts/divorce/notification/CommonContent.java

```diff
@@ -35,23 +39,36 @@
     @Autowired
     private EmailTemplatesConfig emailTemplatesConfig;
 
+    public Map<String, String> templateVarsForApplicant(final CaseData caseData, Applicant applicant) {
+        Map<String, String> templateVars = templateVarsFor(caseData);
+
+        Applicant partnerApplicant = caseData.isApplicant1(applicant) ? caseData.getApplicant2() : caseData.getApplicant1();
```

Comment:
rather than having isApplicant1 method on caseData, I would prefer this method took three arguments:

```
public Map<String, String> templateVarsForApplicant(final CaseData caseData, Applicant applicant, Applicant partner)
```

It's easier to work out the context from the calling code rather than inside here